### DISCUSSION TITLE
Creating more than one 'back' connections.

### DISCUSSION THREAD

**Discussion by gmyles61p** [opendcim/openDCIM#1484] [Q&A]
I have many inter-building fiber links that require establishing in OpenDcim. They have been entered at each end using the "Generic - FO TRAY 1U (12 PORT)" template. When I establish 'back' connections from the 'source' to the 'destination', as per the patch panel youtube video, all 12 connections are established between the two. My question is how do I establish differing 'back' connections when I have, for example, inter-building links where ports 1 - 6 of the FO Tray goes to one building and ports 7 - 12 go to a different building. I also have some that go to three different buildings. Thanks.

**Comment by sfeldmann-uwh** [comment]
You can map them individually, but that could be time consuming.  I would suggest that you create "chassis" panels that would then have 2 or more slots filled with 6-port panels even if they are on the same physical FAP.  You could then map 6 connections to the corresponding back side.

**Reply by gmyles61p** [reply]
I am using the 'chassis' patch panel to good effect now.

Thanks very much for the tip.
